framework_tvb/tvb/adapters/exporters/tvb_export.py

```diff
@@ -81,9 +82,20 @@ def export(self, data, export_folder, project):
             return download_file_name, zip_file, True
 
         else:
-            data_file = h5.path_for_stored_index(data)
-            return download_file_name, data_file, False
+            data_file = self.copy_dt_to_export_folder(data, export_folder)
+            return None, data_file, True
 
+    def copy_dt_to_export_folder(self, data, data_export_folder):
+        data_path = h5.path_for_stored_index(data)
+        file_destination = os.path.join(data_export_folder, os.path.basename(data_path))
+        if not os.path.exists(file_destination):
+            FilesHelper().copy_file(data_path, file_destination)
+
+        with H5File.from_file(file_destination) as dest_file:
+            if 'parent_burst' in dest_file.storage_manager.get_metadata():
+                dest_file.storage_manager.remove_metadata('parent_burst')
```

Comment:
Could this code be simplified to avoid accessing `storage_manager` directly from here by adding a `remove_metadata_param` method on `H5File`, and handling inside that method the `if` condition? We won't need to open the file here either. Would be similar to the `H5File.get_metadata_param` static method.